### PR TITLE
DOPS-2614 Prepare jest for coverage, set up sonar,dojo

### PR DESCRIPTION
**Pull request by C4tWithShell**
1. Prepare jest config (no unit test)
2. Connect to the DefectDojo

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -7,8 +7,9 @@ def pipeline = new org.js.AppPipeline(
     dockerImageName:    'sora2/subquery',
     buildDockerImage:   'docker.soramitsu.co.jp/build-tools/node:14-ubuntu',
     sonarProjectName:   'sora-subquery',
-    sonarProjectKey:    'jp.co.soramitsu:sora-subquery',
+    sonarProjectKey:    'sora:sora-subquery',
```
Comment (baotn166) [inline Jenkinsfile:10]: do we have a format for sona project key?

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "tsc -b",
     "prepack": "rm -rf dist && npm build",
-    "test": "jest",
+    "test": "jest --coverage",
```
Comment (baotn166) [inline package.json:9]: do we need to set config file like `jest --config=./jest.config.js`?

Comment (baotn166) [reply package.json:9]: I've faced an issue before when running a test with jest, it doesn't exit after run. Not sure if you face it

File: Jenkinsfile (hunk 1 above)
Comment (C4tWithShell) [reply Jenkinsfile:10]: Yes - https://sonar.soramitsu.co.jp/admin/permission_templates

File: package.json (hunk 2 above)
Comment (C4tWithShell) [reply package.json:9]: We don't need `--config` flag - it is default option. I did not faced this issue with jest but I will test

**Review by Cre-eD — APPROVED** [review]

**Review by wpi86 — APPROVED** [review]

**Review by BAStos525 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (C4tWithShell) [reply package.json:9]: > I've faced an issue before when running a test with jest, it doesn't exit after run. Not sure if you face it

Okay, I run it 20th times for 2 different projects but did not face this issue

**Pull request merged**